test/ex_machina/ecto_test.exs

```diff
@@ -42,4 +42,22 @@ defmodule ExMachina.EctoTest do
       TestFactory.params_for(:user_map)
     end
   end
+
+  test "params_with_assocs/2 inserts belongs_tos that are set by the factory" do
+    assert Enum.member?(ExMachina.Article.__schema__(:associations), :editor)
```

Comment:
Sorry I wasn't that clear about that. I meant that this test relies on `editor` being something, but it's not clear in this test what it is. You'd have to look at the schema to realize that it's defined, but not being set in the factory. So there is no way to look at this test and realize that `editor` is important in any way. So I think that you could assert that it's on the schema and give it an explanatory name like `defined_on_schema?(Article, :editor)` or something